ember-app/app/templates/issue/reference.hbs

```diff
@@ -11,6 +11,9 @@
      <span class="commit-message"> 
        <span class="commit-link">-o-</span> {{hb-avatar user=model.actor width=16 height=16 }} 
        {{view.message}}
+       {{#unless view.message}}
+         <span class="empty">No commit message</span>
+       {{/unless}}
```

Comment:
Figured it out - turns out [`Ember.String.htmlSafe()`](http://emberjs.com/api/classes/Ember.String.html#method_htmlSafe) returns a [`Handlebars.SafeString`](http://handlebarsjs.com/reference.html#base-SafeString), and an empty `SafeString` is not falsey.